main.js

```diff
@@ -109,30 +109,38 @@ function SquishyList(rootEl, opts) {
 		return hiddenItemEls;
 	}
 
-	function getRemainingItems() {
-		return allItemEls.filter(function(el) {
-			return hiddenItemEls.indexOf(el) === -1;
-		});
-	}
-
-	function squish() {
-		showAllItems();
-		if (doesContentFit()) {
-			hideEl(moreEl);
-		} else {
-			for (var p = prioritySortedItemEls.length - 1; p >= 0; p--) {
-				hideItems(prioritySortedItemEls[p]);
-				if ((getVisibleContentWidth() + moreWidth) <= rootEl.clientWidth) {
-					showEl(moreEl);
-					break;
-				}
-			}
-		}
-		dispatchCustomEvent('oSquishyList.change', {
-			hiddenItems: getHiddenItems(),
-			remainingItems: getRemainingItems()
-		});
-	}
+    function getRemainingItems() {
+        return allItemEls.filter(function(el) {
+            return (hiddenItemEls && hiddenItemEls.indexOf(el) === -1);
+        });
+    }
+
+    function squish() {
+        let previousHidden = getHiddenItems();
```

Comment:
you can't use let though, you're merging to v1, which isn't an es6 compatible branch